test/ford.hpp

```diff
@@ -47,9 +47,15 @@
 
 // An improved async, that doesn't block
 template <class Function>
+#if HIP_PACKAGE_VERSION_FLAT >= 5004000000ULL
```

Comment:
@atamazov  @JehandadKhan  @pfultz2 need you to take a look at this change too.

Reply:
No objections provided that it works fine ;) Any doubts?